pypeit/core/arc.py

```diff
@@ -562,12 +565,27 @@ def detect_lines(censpec, nfitpix=5, sigdetect = 5.0, FWHM = 10.0, cont_samp = 3
     tampl = np.interp(pixt, xrng, arc)
     #         width is fine & width > 0.0 & width < FWHM/2.35 &  center positive  &  center on detector
     #        & amplitude not nonlinear
-    good = (np.invert(np.isnan(twid))) & (twid > 0.0) & (twid < FWHM/2.35) & (tcent > 0.0) & (tcent < xrng[-1]) & \
+    good = (np.invert(np.isnan(twid))) & (twid > 0.0) & (twid < fwhm/2.35) & (tcent > 0.0) & (tcent < xrng[-1]) & \
            (tampl_true < nonlinear_counts)
     ww = np.where(good)
     # Compute the significance of each line, set the significance of bad lines to be -1
     nsig = (tampl - med)/stddev
 
+    if nfind is not None:
```

Comment:
Please comment what this snipped is doing.

Reply:
Done.